### PR TITLE
Add Documentation for Gitpod

### LINKED ISSUE #229: Gitpod usage documentation
Thanks to @MondalSayantan, it is possible to use Gitpod to contribute to the project (see #209). 
However, this is not documented in the CONTRIBUTING.adoc file.

# Requirements

* [ ] how to use gitpod is documented in the contributing guide

### PR DESCRIPTION
**Pull request by MondalSayantan**
### Description
Closes #229 

### Requirements
- [x] how to use gitpod is documented in the contributing guide

### Submitter checklist

- [x] If the issue exists, it is well described and linked in the description
- [x] The description of this pull request is detailed and explain why this pull request is needed
- [x] The changeset is on a specific branch
  - `feature/` for new feature, or improvements
  - `fix/` for bug fixes
  - `docs/` for any documentation changes
- [x] If required, the documentation has been updated
- [ ] There is automated tests to cover the code change / addition
    - If there is no test, include an explanation why in the description
- [ ] Run `mvn verify` locally and all tests are passing successfully
  - It is OK to create a pull request which has failing tests if it is created as a draft, is to fix a bug and the first commit is the test to prove the existence of the bug.
- [x] There is no new warnings (checkstyle nor spotbugs) on the code

### CODE REVIEW

File: docs/CONTRIBUTING.adoc [hunk 1]
```diff
@@ -133,6 +133,12 @@ If you are using Docker you can connect to the database using the following comm
 docker container exec -it plugin-health-scoring-db-1 psql -U <postgres-user-name>
 ----
 
+== Run the application on Gitpod
+
+This project includes a configuration in `.gitpod.yml` to be able to run on Gitpod. The required versions of Java and Maven are installed when starting the Gitpod workspace. Follow the steps to run the application using Docker compose.
```
Comment (alecharp) [inline docs/CONTRIBUTING.adoc:138]: NOTE: Having one sentence per line simplify the correction / comments.

On the last line, could you elaborate a bit more (why would someone need to run the application? etc.)

```suggestion
This project includes a configuration in `.gitpod.yml` to be able to contribute to it on Gitpod.
The required versions of Java and Maven are installed when starting the Gitpod workspace.

Follow the steps to run the application using Docker compose.
```

Comment (MondalSayantan) [reply docs/CONTRIBUTING.adoc]: I have elaborated more in my latest commits and also tried to keep it concise

**Review by alecharp — CHANGES_REQUESTED** [review]
I feel like this is better. 
Just a few tweak and we will be good.

**Comment by MondalSayantan** [comment]
Thank you so much for the review!

**Review by alecharp — APPROVED** [review]

**Pull request merged**